modules/imgproc/perf/perf_warp.cpp

```diff
@@ -49,36 +53,6 @@ PERF_TEST_P( TestWarpAffine, WarpAffine,
 #endif
 }
 
-PERF_TEST_P(TestWarpAffine, DISABLED_WarpAffine_ovx,
```

Comment:
OpenVX support is removed in https://github.com/opencv/opencv/pull/25197. So I remove this test. Also removed perf data from `$opencv_extra/testdata/perf/imgproc.xml`.